src/harness/fourslashImpl.ts

```diff
@@ -1775,7 +1775,7 @@ export class TestState {
             position,
             findInStrings,
             findInComments,
-            providePrefixAndSuffixTextForRename,
+            { providePrefixAndSuffixTextForRename },

```

Comment:
Do we want to add option for quotePreference and test that it takes the preferred quote?

Reply:
Sure.